cvat/apps/engine/models.py

```diff
@@ -437,6 +438,36 @@ class Meta:
     def touch(self) -> None:
         self.save(update_fields=["updated_date"])
 
+class AssignableModel(models.Model):
+    assignee = models.ForeignKey(
+        User, null=True, blank=True, on_delete=models.SET_NULL, related_name="assigned_%(class)ss"
+    )
+    assignee_updated_date = models.DateTimeField(null=True, blank=True, default=None)
+
+    class Meta:
+        abstract = True
+
+    def update_assignee(
+        self,
+        *,
+        assignee_id: int | None = None,
+        updated_date: datetime.datetime | None = None,
+        save: bool = True,
+    ) -> None:
+        self.assignee_id = assignee_id
+        self.assignee_updated_date = updated_date or timezone.now()
+
+        if save:
+            self.save(update_fields=["assignee, assignee_updated_date"])
+
+class AssignableModelWithoutBackRelationship(AssignableModel):
```

Comment:
Why is this class needed? You've used it for `Issue`, but why not let it inherit from `AssignableModel`?